src/test/suite/commands/getAddon.test.ts

```diff
@@ -75,6 +81,9 @@ describe("getAddon.ts", () => {
       },
     });
 
+    const module3 = await import("../../../amo/utils/addonCache");
```

Comment:
If there is more than one `module` it's better to have descriptive names, like `moduleAddonsCache`.  Applies to consts such as `stub3` too but as that's existing code I'll just note it for the next time you touch this tests.

Reply:
I'll do this in the next test refactor